scripts/protocgen.sh

```diff
@@ -73,8 +73,8 @@ google/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types:. \
 done
 
 # generate codec/testdata proto code
-buf protoc -I "proto" -I "third_party/proto" -I "testutil/testdata" --gocosmos_out=plugins=interfacetype+grpc,\
-Mgoogle/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types:. ./testutil/testdata/*.proto
+buf protoc -I "proto" -I "third_party/proto" -I "orm/testdata" --gocosmos_out=plugins=interfacetype+grpc,\
```

Comment:
Running `make proto-gen' command to gen *.proto inside orm/testdata and getting the following [error](https://play.golang.org/p/0yHihITqq27) (((

Reply:
You need to add inside the root `go.mod`:
```
replace github.com/regen-network/regen-ledger/types => ./types
replace github.com/regen-network/regen-ledger/x/data => ./x/data
replace github.com/regen-network/regen-ledger/x/ecocredit => ./x/ecocredit
replace github.com/regen-network/regen-ledger/x/group => ./x/group
```